gluoncv/model_zoo/fcn.py

```diff
@@ -86,7 +89,7 @@ def hybrid_forward(self, F, x):
 
 
 def get_fcn(dataset='pascal_voc', backbone='resnet50', pretrained=False,
-            root='~/.mxnet/models', ctx=cpu(0), **kwargs):
+            root='~/.mxnet/models', ctx=cpu(0), backbone_pretrained=True, **kwargs):
```

Comment:
can you use name `pretrained_base` to maintain consistency with detection models?
https://gluon-cv.mxnet.io/api/model_zoo.html#gluoncv.model_zoo.get_ssd

Reply:
Ah, ok sure. I will make this change.

Reply:
Yes, I will make this fix right away.

Reply:
Fixed.